Reconstruction/reconstruction.tex

```diff
@@ -111,7 +111,7 @@ \subsection{ECAL Superclusters}
 Since the crystals in the endcaps are arranged in an $x$-$y$ grid, clustering here uses fixed $5\times5$ matrices of crystals. 
 After a seed cluster is identified, additional, partially overlapping $5\times5$ matrices are added if their centroid lies within $\abs{\Delta\eta} \le 0.07$ and $\abs{\Delta\phi} \le 0.3$.
 For unconverted photons, both methods produce superclusters that are simple $5\times5$ matrices.
-
+\PH{The above paragraph is quite cryptic. I struggled to figure out of the 5x1 is in $\eta$ or $\phi$. Also, how the iterative choice of the seeding works.  Given that this is an important part of your analysis can you rewrite.}
```

Comment:
attempted to clarify the dimensions of the arrays. the reseeding process is totally undescribed in the official recosntruction papers and I don't want to dig through the code.